core/src/main/scala/com/snowplowanalytics/snowplow/eventgen/protocol/enrichment/Enrichment.scala

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) 2021-present Snowplow Analytics Ltd. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+ */
+package com.snowplowanalytics.snowplow.eventgen.protocol.enrichment
+
+trait Enrichment
```

Comment:
I see that each enrichment extends this trait but I don't see this type used anywhere else?

Reply:
I started with some other design but change it to another during the implementation. It is remain from that, I forgot to remove it. I will remove it 👍